### ISSUE TITLE
ClassCastException on query with orderBy

### ISSUE THREAD

**Issue by vadimbasko** [Haulmont/jmix-graphql#110]
```
{
  scr_CarList(orderBy: {manufacturer: ASC}){
    id
    lastModifiedDate
  }
}
```

```
java.lang.ClassCastException: class java.util.Collections$SingletonList cannot be cast to class java.lang.String (java.util.Collections$SingletonList and java.lang.String are in module java.base of loader 'bootstrap')
	at io.jmix.graphql.datafetcher.EntityQueryDataFetcher.buildOrderBy(EntityQueryDataFetcher.java:153)
	at io.jmix.graphql.datafetcher.EntityQueryDataFetcher.lambda$loadEntities$2(EntityQueryDataFetcher.java:104)

```

**Comment by DartVerder** [comment]
#137

**Comment by vadimbasko** [comment]
Fixed in #137
